crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -1754,6 +1765,23 @@ mod tests {
         provider.commit().unwrap();
     }
 
+    /// Use this instead of [insert_blocks] when inserting hashes and calculating history indices
+    /// for each block is needed
```

Comment:
would add a note on what it does as well, this only explains when to use one over the other (so you have to refer to `insert_blocks`)

Reply:
added an extra note above, lmk if unclear